opencog/reduct/reduct/reduct.h

```diff
@@ -109,7 +110,11 @@ struct logical_reduction
 
 	logical_reduction(const vertex_set &ignore_ops);
 
-	logical_reduction(const HandleSet &ignore_ops);
+	logical_reduction(const HandleSet &ignore_ops)
+			: logical_reduction(set_comp(ignore_ops, [this](const Handle h) {
+		return *to_combo(h).first.begin();
+	}))
+	{}
```

Comment:
Really minor point but it would probably have been better to have this code in `logical_reduction.cc`.